sdk/formrecognizer/test-resources.json

```diff
@@ -155,7 +155,7 @@
         "classifierTrainingSasPropertiesV3": {
             "type": "object",
             "defaultValue": {
-                "canonicalizedResource": "[concat('/blob/', parameters('trainingDataAccount'), '/', parameters('classifierTrainingDataContainer'), '-v3')]",
+                "canonicalizedResource": "[concat('/blob/', parameters('trainingDataAccount'), '/', parameters('classifierTrainingDataContainer'))]",
```

Comment:
The container we use to train classifiers is called `training-data-classifier`, but we were assuming `training-data-classifier-v3` in the test resources file.